.github/workflows/nightly-patch-release.yml

```diff
@@ -40,7 +40,4 @@ jobs:
     strategy:
       matrix: ${{ fromJson(needs.generate-version-matrix.outputs.matrix) }}
       fail-fast: false
-    uses: aws/sagemaker-distribution/.github/workflows/nightly-build-image.yml@main
```

Comment:
If all we need to do is build the one version, it might be best to just have the nightly-build-image workflow. Triggered manually/by schedule.
For the public release, the separate patch file was mostly to generate all the necessary patch versions.

Reply:
Removed the file in the latest commit